docs/resources/sobject.md

```diff
@@ -100,7 +186,7 @@ resource "dsm_sobject" "sobject" {
       description = "The policy document. This is a JSON formatted string."
       default = <<-EOF 
               {
-               "description":"Credit card"
+               "description":"Credit card,"
```

Comment:
It seems this comma should be swapped `,"` -> `",`

Reply:
Changed